tools/stores_l10n/add_new_product.md

```diff
@@ -0,0 +1,261 @@
+* There are no keywords in Android, short description is not available for iOS products.
+
+## Tests
+
+It’s not mandatory to add new tests, since they normally cover functionalities and not data. Adding a simple API test in `tests/functional/api.php` is highly suggested. For example:
+
+```PHP
+$paths = [
+...
+    ['v1/focus_android/done/release/', 200, false],
+...
+```
+
+Some tests will need to be updated, for example `testGetSupportedProducts` in [tests/units/Store/Project.php](https://github.com/mozilla-l10n/stores_l10n/blob/master/tests/units/Store/Project.php). In case, unit tests are run as part of the automation when opening a pull request.
+
+## Track .lang files in Langchecker
+
+The Stores web app is going to use .lang files from the [appstores repository](https://github.com/mozilla-l10n/appstores). This document won’t go into much detail about these configuration files, if you’re not familiar with Langchecker you should read [this document](/tools/webdashboard/add_new_file-md) first.
+
+In the configuration two .lang file are associated to Focus for Android:
+* `focus_android/description_release.lang` for the main content.
+* `focus_android/screenshots_v1.lang` for screenshots.
+
+First you need to create a new `$focus_android_locales` array in `app/config/stores_locales.inc.php`.
+
+```PHP
+// Locales working on Focus for Android (from stores_l10n app)
+$cache_id = 'focus_android_locales';
+if (! $focus_android_locales = Cache::getKey($cache_id, 60 * 60)) {
+    $focus_android_locales = $json_object
+        ->setURI(STORES_L10N . 'focus_android/supportedlocales/release')
+        ->fetchContent();
+    Cache::setKey($cache_id, $focus_android_locales);
+}
+```
+
+This code retrieves, via API from the Stores web app, the list of locales supported for Focus for Android release. Then:
+
+```PHP
+$focus_android_store = array_intersect($focus_android_locales, $google_play_locales);
+```
+
+This intersects the list of locales shipping in Focus for Android with the list of locales supported in Play Store, and stores the resulting array in `$focus_android_store`.
+
+Then you need to start tracking the new files in `app/config/sources.inc.php`:
+
+```PHP
+$appstores_lang = [
+    'focus_android/description_release.lang' => [
+        'supported_locales' => $focus_android_store,
+    ],
+    'focus_android/screenshots_v1.lang' => [
+        'supported_locales' => [
+            'es-ES', 'id', 'pt-BR', 'ru',
+        ],
+    ],
+...
+];
+```
+
+Once the configuration is complete, you’ll need to create the actual .lang files and add them to the appstores repository with `lang_update`.
+
+## Testing
+
+When all tools and repositories are ready, make sure to test everything locally and check PHP logs before opening a pull request. It’s strongly recommended to use a virtual machine set up following these [instructions](/config/setup_l10ndrivers_vm.md) running all needed tools.
```

Comment:
Maybe add that an easy way to test is "translating" one locale, and check the views, APIs…